src/ophys_etl/transforms/suite2p_wrapper.py

```diff
@@ -0,0 +1,219 @@
+import argschema
+import suite2p
+import pathlib
+import marshmallow as mm
+import h5py
+import tempfile
+from typing import List
+import datetime
+import shutil
+
+
+class Suite2PWrapperException(Exception):
+    pass
+
+
+class Suite2PWrapperSchema(argschema.ArgSchema):
+    # Options for running s2p. One list can be found at:
+    # https://github.com/MouseLand/suite2p/blob/master/suite2p/run_s2p.py
+
+    # s2p IO settings (file paths)
+    h5py = argschema.fields.InputFile(
+            required=True,
+            description="Path to 2P motion corrected hdf5 data path.")
+    h5py_key = argschema.fields.Str(
+            required=False,
+            missing="data",
+            default="data",
+            description="hdf5 file key for motion corrected dataset.")
+    # s2p registration settings
+    do_registration = argschema.fields.Int(
+            default=0,
+            description="0 skips registration, as we are doing our own.")
+    # s2p cell detection settings
+    roidetect = argschema.fields.Bool(
+            default=True,
+            description="Whether or not to run ROI extraction.")
+    sparse_mode = argschema.fields.Bool(
+            default=True,
+            description="Whether or not to run 'sparse mode'.")
+    diameter = argschema.fields.Int(
+            default=12,
+            description=("If not sparse_mode, use diameter (presumably "
```

Comment:
might want to clarify 'presumably' before putting it in the description, if possible to clarify.

Reply:
this is copied from Suite2P. Added something to that effect.